### DISCUSSION TITLE
adding sticker pack name and author

### DISCUSSION THREAD

**Discussion by orangbilangciperipampam** [tulir/whatsmeow#408] [WhatsApp protocol Q&A] [answered]
StickerMessage: &waProto.StickerMessage{
			Url:           proto.String(uploaddata.URL),
			DirectPath:    proto.String(uploaddata.DirectPath),
			MediaKey:      uploaddata.MediaKey,
			Mimetype:      proto.String(http.DetectContentType(webpdatsa)),
			FileEncSha256: uploaddata.FileEncSHA256,
			FileSha256:    uploaddata.FileSHA256,
			FileLength:    proto.Uint64(uint64(len(dataimage))),
			ContextInfo: &waProto.ContextInfo{
				StanzaId:      &helpers.MsgController.Info.ID,
				Participant:   proto.String(helpers.MsgController.Info.Sender.String()),
				QuotedMessage: helpers.MsgController.Message,
				RemoteJid:     proto.String(helpers.MsgController.Info.Chat.String()),
			},

how to add sticker author
and sticker pack name ?

**Comment by akshettrj** [comment, marked as answer]
You need to add EXIF data to the sticker. Something like this: https://github.com/akshettrj/watgbridge/blob/main/utils/stickers.go#L173

**Reply by orangbilangciperipampam** [reply]
thanks

already solved
using webpmux exec to set the exif
